### ISSUE TITLE
Feature Request: Store Node ID in messages DB instead of sender short name

### ISSUE THREAD

**Issue by Bamorph** [pdxlocations/Meshtastic-MQTT-Connect#44]
In the database i think it would be better to save the node if "!abcdef123" instead of the short name, and when we load the messages from database, we cross reference the nodeinfo to set the short name.

**Comment by pdxlocations** [comment]
Makes sense to save the node id from a database perspective.  If someone changes their short name, the current model is stuck with the original name but with your approach, it would update to the new name when you load the db.
